standard/vod_link.lua

```diff
@@ -48,4 +49,4 @@ function VodLink.display(args)
 		:wikitext('[[File:' .. fileName .. '|link=' .. link .. ']]')
 end
 
-return Class.export(VodLink, {frameOnly = true})
+return Class.export(VodLink, {frameOnly = true})
```

Comment:
```suggestion
return Class.export(VodLink, {frameOnly = true})

```

Reply:
wut

Reply:
https://github.com/Liquipedia/Lua-Modules/pull/1961/files#diff-80225ce5b1709fd14e1f1bf6a35079e80e1e6cb7e030f9643b5c8be2ca674635R52

You removed the EOF new line

Reply:
thought it should be removed xd